### PR TITLE
fix(curriculum): update video ids

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This updates video ids for:

- Understanding Computer, internet and tooling basics
still to update:
    - [x] 672ab82c1a9bbd0e3aabc39d
    - [x] 672ab83c4297910eade53c2e
    - [x] 672ab849aa1ef70eefd29364
    - [x] 672ab8573f32480f192aaae1

- Working with functions
still to update:
    - [x] 672d269da46786225e3fe3fd TRANSCRIPT ADDED
    - [x] 673284d5e52ef81a2169b097 TRANSCRIPT ADDED
    - [x] 673284e7244c0c1a649121b9 TRANSCRIPT ADDED

- Understanding core JavaScript Fundamentals
still to update:
    - [x] 6732c6ab21031b60d2b0c999
    - [x] 6732c6ba2ea42b610b9f9ce1
    - [x] 6732c6c72d3738614e1230a2
    - [x] 6732c6e281c14a61c4858361
    - [x] 6732c69d82814160951b1aa7

- The var keyword and Hoisting
still to update:
    - [x]  67335f45489c5a11b71d0ed5

- What is a module in JavaScript and how can you import/export modules in your program?
still to update:
    - [x]  67329fd6ad99c75d4a4b74e4

- Working with Higher Order Functions and callbacks
still to update:
    - [x] 673362a34edda41dedf87623 TRANSCRIPT ADDED
    - [x] 673362b3f763ae1e38e17df7
    - [x] 673362be2f70c21e65bc5459
    - [x] 673362d7f94d551edb532d24
    - [x] 673362e43d57b51f1ad2d466

- Debugging techniques
still to update:
    - [x] 6733becf4b0c353553b9bfa4
    - [x]  6733befb703ca6361da3755b

- Working with maps and sets

- Understanding form validation
    - [x] 6733d3ab69e94b7df7ee91b0 TRANSCRIPT ADDED

- Understanding recursion and the call stack
    - [x] 6733b02d1e556005a544c5e3 TRANSCRIPT ADDED

### CODE REVIEW

**Comment by Dario-DC** [comment]
> It looks like we have a video now for HTML entities
> 
> https://www.youtube.com/watch?v=mVdIq0H_B4E

It seems it's been already updated https://github.com/freeCodeCamp/freeCodeCamp/blob/397c4a5372a34e69dd8278a21d6fe041d597a01e/curriculum/challenges/english/25-front-end-development/lecture-html-fundamentals/67083868d5fdcb17bf8c14bd.md?plain=1#L5

**Comment by jdwilkin4** [comment]
Some of the links in the computer basics module were updated so the ids will need to updated here

File: curriculum/challenges/english/25-front-end-development/lecture-understanding-computer-internet-and-tooling-basics/672ab82c1a9bbd0e3aabc39d.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 id: 672ab82c1a9bbd0e3aabc39d
 title: How Can You Effectively Work With Your Keyboard, Mouse, and Other Pointing Devices
 challengeType: 11
-videoId: nVAaxZ34khk
+videoId: vuXNEz8zmoA
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/lecture-understanding-computer-internet-and-tooling-basics/672ab82c1a9bbd0e3aabc39d.md:5]:
```suggestion
videoId: AcTxNEVccxc
```

**Comment by Dario-DC** [comment]
> Some of the links in the computer basics module were updated so the ids will need to updated here

I found only one among those updated in this PR @jdwilkin4 . Do you mean that some of those already added to main needs update?

**Comment by jdwilkin4** [comment]
It looks like you added the updated one already 👍

**Review by Nirajn2311 — APPROVED** [review]

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**